packages/suite/src/support/extraDependencies.ts

```diff
@@ -25,7 +25,7 @@ const connectInitSettings = {
     transportReconnect: true,
     debug: false,
     popup: false,
-    webusb: isWeb(),
+    webusb: isWeb(), // deprecated, use transports
```

Comment:
why not to use it right away?

Reply:
fixed up in https://github.com/trezor/trezor-suite/pull/7411/commits/96a0030d4d838643c0afd7a94831b93794170f64
this param should not be needed at all, since connect now has defaults we want to use